engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -59,15 +76,15 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
           stay()
       }
     case Event(f: JobStoreReadFailure, NoData) =>
-      log.error(f.reason, "Error reading from JobStore for " + jobKey)
+      log.error("{}: Error reading from JobStore: {}", tag, f.reason)
```

Comment:
Shouldn't this be:

```
log.error(f.reason, "{}: Error reading from JobStore", tag)
```
to get the stack trace?  Unless you don't want the stack trace, which maybe you don't.